docs/release-notes/latest.md

```diff
@@ -9,100 +9,100 @@ tags:
-    * Semgrep Cloud Platform
-    * Semgrep Code
-    * Semrep Supply Chain
-    * Semgrep Assistant (beta)
-    * Semgrep documentation and knowledge base
-* Refer to **Semgrep OSS** release notes in [<i class="fas fa-external-link fa-xs"></i> Semgrep GitHub > Releases](https://github.com/returntocorp/semgrep/releases/) as the source of truth for OSS releases.
-:::
-
-## Private beta sign-ups
-
-* **Semgrep Secrets** is a code scanner that detects exposed API keys, passwords, and other credentials. Sign up for the private beta by filling out the [<i class="fas fa-external-link fa-xs"></i> Semgrep Secrets Beta](https://get.semgrep.dev/secrets-beta-request.html) form.
-* **Semgrep Supply Chain SBOM (software bill of materials)** enables you to export a list of dependencies in the CycloneDX 1.4 XML/JSON format. Sign up for the private beta by filling out the [<i class="fas fa-external-link fa-xs"></i> SSC SBOM Export](https://get.semgrep.dev/SBOM-Export-private-beta.htm) form.
+# October 2023 release notes
 
 ## 🔧 Semgrep OSS Engine
 
-* The following versions of Semgrep OSS Engine were released in September 2023:
-  * [<i class="fas fa-external-link fa-xs"></i> 1.38.0](https://github.com/returntocorp/semgrep/releases/tag/v1.38.0)
-  * [<i class="fas fa-external-link fa-xs"></i> 1.39.0](https://github.com/returntocorp/semgrep/releases/tag/v1.39.0)
-  * [<i class="fas fa-external-link fa-xs"></i> 1.40.0](https://github.com/returntocorp/semgrep/releases/tag/v1.40.0)
-  * [<i class="fas fa-external-link fa-xs"></i> 1.41.0](https://github.com/returntocorp/semgrep/releases/tag/v1.41.0)
-  * [<i class="fas fa-external-link fa-xs"></i> 1.42.0](https://github.com/returntocorp/semgrep/releases/tag/v1.42.0)
+- The following versions of Semgrep OSS Engine were released in October 2023:
+  - [<i class="fas fa-external-link fa-xs"></i> 1.43.0](https://github.com/returntocorp/semgrep/releases/tag/v1.43.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.44.0](https://github.com/returntocorp/semgrep/releases/tag/v1.44.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.45.0](https://github.com/returntocorp/semgrep/releases/tag/v1.45.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.46.0](https://github.com/returntocorp/semgrep/releases/tag/v1.46.0)
 
 ## 🌐 Semgrep Cloud Platform
 
 ### Added
+- Added a button to **Remove** source code manager (SCM) apps. This is helpful when you may have a malconfigured SCM app, such as GitHub's `semgrep-app`, and want to reinstall it. <!--(10688)--> To remove an SCM, click **<i class="fa-solid fa-gear"></i> Settings > Source code managers**.
+![Remove your source code manager](/img/settings-scm-remove.png)
+- Added Semgrep Assistant to the new onboarding flow. <!--(10716) -->
+- **OpenAPI:** Renamed instances of r2c to Semgrep. <!--(10685) -->
+- **Projects page:** Added a new column to display a Semgrep Secrets counter. This counter counts all secrets regardless of validation state. <!--(10588)-->
```

Comment:
I wonder if it makes sense to put this under Secrets?

Reply:
ACKnowledged - will move it.